Client/src/Components/Account.tsx

```diff
@@ -0,0 +1,20 @@
+import { account } from "../interface";
+import LoginGoogle from "./LoginGoogle";
+
+const Account = ({ Title }: account) => {
+  return (
+    <>
+          <h1 className="font-bold text-3xl">TaskMo4a</h1>
+          <div className="my-10">
+            <p className="text-xl">{Title} your account</p>
+            <p>
+              Unlock exclusive perks - Sign up now for early access and special
+              offers!
```

Comment:
Can we have any kind of localization? What if I want to have all texts in Arabic and English? and it switches automatically based on region?

Reply:
i can't understand this,
but i found this Text in Template that i am trying to copy, can you understand me?

Reply:
Please research Web localization, and let's discuss it after

Reply:
Okay i will research about this topic